pkgs/applications/science/biology/deeptools/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, pytest
+, numpy
+, numpydoc
+, scipy
+, py2bit
+, pybigwig
+, pysam
+, matplotlib
+, plotly
+, deeptoolsintervals
+}:
+buildPythonApplication rec {
+  pname = "deepTools";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08p36p9ncj5s8qf1r7h83x4rnmi63l3yk6mnr3wgpg2qgvwl0hji";
+  };
+
+  propagatedBuildInputs = [ numpy numpydoc scipy py2bit pybigwig pysam matplotlib plotly deeptoolsintervals ];
+
+  checkInputs = [
+    pytest numpy numpydoc scipy py2bit pybigwig pysam matplotlib plotly deeptoolsintervals
```

propagatedBuildInputs will also appear in the checkPhase
```suggestion
    pytest
```